Source/Fuse.Auth/BiometricModule.uno

```diff
@@ -14,7 +14,7 @@ namespace Fuse
 		## Example
 
 			The following example shows how to use it:
-
+			```javascript
```

Comment:
`xml` is better here I think?

Reply:
In these specific cases, JS is better for visual highlight, because we have more (or almost equal) code lines then the markup tags. In case of use `xml` we lost the js highlight

Reply:
Cool, that's a good point. In that case it looks like writing `js` is more popular than `javascript`.

Reply:
They are equal for the highlighting engine. We can use both